src/commandHandlers/bio.ts

```diff
@@ -49,6 +49,26 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         }
     }
 
+    const updateBio = async (data: string | object, entry: string, msg: Message) => {
```

Comment:
I think you have access to these fields in the function scope, so maybe we could remove these parameters since they are not necessary (unless we move this function out of the scope).
What do you think @AllanRegush ?

Reply:
I'll give it shot